test/data/corrective-invoice.json

```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "https://gobl.org/draft-0/envelope",
+  "head": {
+    "uuid": "d05c7988-e145-11ee-bef0-52f0b4d06139",
+    "dig": {
+      "alg": "sha256",
+      "val": "0630d41061873a3b8bf650cbfd204a57c6157210d8a6f49c4a4ee31338b43e1f"
+    },
+    "draft": true
+  },
+  "doc": {
+    "$schema": "https://gobl.org/draft-0/bill/invoice",
+    "type": "corrective",
```

Comment:
I'd expect this to be `credit-note` and the rest of the numbers in the document to not be negative.